### ISSUE TITLE
 The following site was not found in the configuration file: 'umbraco-sample-f43b72dc-5c06-425b-a7cf-e83ad7e60765'

### ISSUE THREAD

**Issue by emeria** [umbraco/UmbracoDocs#1848]
I followed the steps exactly but got the above error when running iisexpress with ctrl+F5.

***This is the page with issues: https://github.com/umbraco/UmbracoDocs/edit/master/Getting-Started/Setup/Install/index.md***

**Comment by sofietoft** [comment]
Hi @emeria 
Sorry to hear you're running into some trouble when installing Umbraco!

I've tried the steps, but cannot reproduce the error here.
Could you give it another shot?
And if you still run into the same, please give me some more details on the exact steps you're following when installing Umbraco! 😁 
/Sofie

**Comment by emeria** [comment]
I followed these steps:
1. Download and install Visual Studio Code
2. Download and install IIS Express (Optional as IIS Express for VSCode will install it)
3. Download and unzip Umbraco (UmbracoCms.8.1.5.zip)
4. Install the IIS Express Extension for VS Code
5. Open the unzipped root folder in VS Code
6. Run the website with the IIS Express Extension(CTRL+F5)

Here is my folder that I opened in VS Code and tried to run with Ctrl+F5:
![image](https://user-images.githubusercontent.com/2154992/65811466-a7e23d80-e186-11e9-877e-e26fcd569696.png)
